.github/workflows/lint.yaml

```diff
@@ -18,18 +17,14 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install uv
-        uses: astral-sh/setup-uv@v3
-        with:
-          enable-cache: true
-          cache-dependency-glob: pyproject.toml
+        uses: astral-sh/setup-uv@v4
 
       - name: Setup Python
-        uses: actions/setup-python@v5
-        with:
-          python-version-file: pyproject.toml
+        shell: bash
+        run: uv venv
```

Comment:
This is actually faster than using `setup-python` and isolates things in a virtual env. One downside is that it just installs the latest Python version by default (3.13 currently). If we want to pin a version we could add one to this command or add a `.python-version` file to the root of our project.

Reply:
**[Praise]** I like this! Nice that it's faster, and I also think it's helpful that it more closely matches the way we use uv for development.